### PR TITLE
feat: restore changes from #701 in separate file

### PR DESCRIPTION
**Pull request by ZyanKLee**
this got lost during work on #720 

to avoid further conflicts between fixes for mopidy-2.3.1 and conversion to python3/mopidy3, I duplicated the installer script.

### CODE REVIEW

**Comment by ZyanKLee** [comment]
@mikebarkmin please have a look at this file and see if it resembles your changes from #701 
@MiczFlor PRs targeting python3 should be adapted to patch `scripts/installscripts/buster-install-mopidy3.sh` instead of the default script.

File: scripts/installscripts/buster-install-mopidy3.sh [hunk 1]
```diff
@@ -0,0 +1,859 @@
...
+
+# power management of wifi: switch off to avoid disconnecting
+sudo iwconfig wlan0 power off
+
+# Install required packages
+sudo apt-get update
+sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages install libspotify-dev apt-transport-https samba samba-common-bin python-dev python-pip gcc raspberrypi-kernel-headers lighttpd php7.3-common php7.3-cgi php7.3 php7.3-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero resolvconf spi-tools python-spidev python3-spidev
+
+# use python3.7 as default
+sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
```
Comment (kingosticks) [inline scripts/installscripts/buster-install-mopidy3.sh]: Why do you need this? Python 3.7 *is* already the default Python version in Buster.

Comment (kingosticks) [reply scripts/installscripts/buster-install-mopidy3.sh]: If the user changed it, it was likely for a reason. Maybe they wanted Pythoni 3.8 and you've just undone that and broken something else unncessarily. Personally I think you'd be better of checking the version of Python >= 3.7 (at the start) and erroring out if not.

Comment (ZyanKLee) [reply scripts/installscripts/buster-install-mopidy3.sh]: that sounds like a good idea - would you care to create a PR with that change?

Comment (kingosticks) [reply scripts/installscripts/buster-install-mopidy3.sh]: Sure. I don't use this project so I can't test it directly but I'll do my best when I get some spare moments later.

**Pull request merged**